src/FastBoard.cpp

```diff
@@ -565,6 +581,22 @@ void FastBoard::display_liberties(int lastmove) {
     myprintf("\n\n");
 }
 
+void FastBoard::print_column_labels(int size, std::string padding) {
+	myprintf("   ");
```

Comment:
nit: inconsistent indentation with the rest of the program

padding isn't change so it can be passed by const reference.

Reply:
Added the const qualifier.